lib/Controller/RoomController.php

```diff
@@ -771,9 +773,9 @@ public function removeGuestFromRoom($token, $participant) {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
-		if ($targetParticipant->getParticipantType() !== Participant::GUEST) {
-			return new DataResponse([], Http::STATUS_FORBIDDEN);
-		}
+		//if ($targetParticipant->getParticipantType() !== Participant::GUEST) {
```

Comment:
I implemented guest moderators in https://github.com/nextcloud/spreed/pull/1078 so this should not be needed anymore.